Domain/sdLitica.Analytics/AnalyticsOperation.cs

```diff
@@ -0,0 +1,48 @@
+﻿using sdLitica.Entities.Abstractions;
+using System;
+
+namespace sdLitica.Analytics
+{
+    //  At this moment, also used as its model in requests, messages. 
+
+    /// <summary>
+    /// Entity for metadata of analytics operation.
+    /// </summary>
+    public class AnalyticsOperation: Entity
+    {
+
+        public AnalyticsOperation()
+        {
+        }
+
+        public void SetId()
+        {
+            Id = Guid.NewGuid();
+        }
+
+
+        public Guid Id { get; set; } // should have 'protected set'
+
+        /// <summary>
+        /// Name of operation to be performed. 
+        /// </summary>
+        public string OpName { get; set; } // should have 'protected set'
+
+        /// <summary>
+        /// Id of time-series on which operation is performed
+        /// </summary>
+        public string TsId { get; set; }
```

Comment:
There is no problem with this name. But, for readability, we can use longer, clear, and meaningful names such as 'TimeSeriesId'. Also, the compiler will change it during compilation, so no issues in the size of binaries.